optuna/logging.py

```diff
@@ -102,6 +102,37 @@ def get_verbosity() -> int:
 def set_verbosity(verbosity: int) -> None:
     """Set the level for the Optuna's root logger.
 
+    Example:
+
+        Set the logging level ``optuna.logging.WARNING``.
+
+        .. testsetup::
+
+            def objective(trial):
+                x = trial.suggest_int("x", -10, 10)
+                return x ** 2
+
+        .. testcode::
+
+            import optuna
+
+            # There are INFO level logs.
+            study = optuna.create_study()
+            study.optimize(objective, n_trials=10)
+            # [I 2021-10-31 02:59:35,088] Trial 0 finished with value: 16.0 ...
+            # [I 2021-10-31 02:59:35,091] Trial 1 finished with value: 1.0 ...
+            # [I 2021-10-31 02:59:35,096] Trial 2 finished with value: 1.0 ...
+
+            # Set the logging level WARNING.
+            # There are INFO level logs.
+            optuna.logging.set_verbosity(optuna.logging.WARNING)
+            study.optimize(objective, n_trials=10)
+
+        .. testcleanup::
+
+            optuna.logging.set_verbosity(optuna.logging.INFO)
```

Comment:
Without this clean-up task, do other tests fail?

Reply:
Yes. If you want to check, please see [these logs](https://github.com/optuna/optuna/runs/4058665366?check_suite_focus=true#step:6:6707).

Reply:
Thanks! That means the doc tests share the logging settings. I was not aware of it.